### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by samilaaroussi**
Travis : Rake, RVM, Gemfile

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: ruby
+
+script: bundle exec rake test
```
Comment (rjollet) [inline .travis.yml]: rake spec I think

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
```
Comment (rjollet) [inline .travis.yml]: I think you also need to say which branch you want to test

**Review by rjollet — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (rjollet) [reply .travis.yml]: https://github.com/ISS-SOA/facegroup/blob/master/.travis.yml

Comment (samilaaroussi) [reply .travis.yml]: Okay

**Comment by samilaaroussi** [comment]
Can you check my update ?

File: .travis.yml [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
 rvm:
   - 2.0.0
   - 2.1.10
   - 2.2.5
   - 2.3.1
   - ruby-head
+branches:
+  only:
+    - master
+    - travis
```
Comment (rjollet) [inline .travis.yml]: only master is enough or you can add develop

**Review by rjollet — APPROVED** [review]

**Pull request merged**